cmd/sonobuoy/app/args.go

```diff
@@ -466,10 +466,16 @@ func (m *Mode) Set(str string) error {
 		if err := m.env.Set(fmt.Sprintf("e2e.E2E_FOCUS=%v", quickFocus)); err != nil {
 			return fmt.Errorf("failed to set flag with value %v", str)
 		}
+		if err := m.env.Set("e2e.E2E_SKIP"); err != nil {
```

Comment:
Actual code change here and right below; just set remove those values if we don't want them.

Confirmed that setting them this way (without the `=foo` ending) will just remove the value.